Add framework-tool-tui (#191)
ratatui/awesome-ratatui: README.md

- [dua-cli](https://github.com/Byron/dua-cli) - View disk space usage and delete unwanted data, fast.
- [ducker](https://github.com/robertpsoane/ducker) - A terminal app for managing Docker containers, inspired by K9s.
- [erldash](https://github.com/sile/erldash) - A simple, terminal-based Erlang dashboard.

> - [framework-tool-tui](https://github.com/grouzen/framework-tool-tui) - TUI for controlling and monitoring Framework Computers hardware built in Rust.

Reviewer: ```suggestion
- [framework-tool-tui](https://github.com/grouzen/framework-tool-tui) - A TUI for controlling and monitoring Framework Computers hardware.
```
